src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -1,67 +1,211 @@
 package org.broadinstitute.dsde.workbench.leonardo.dao
 
+import java.io.{ByteArrayInputStream, File, InputStream, PrintWriter}
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+
 import com.google.api.services.dataproc.model._
 import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
 import org.broadinstitute.dsde.workbench.google.GoogleUtilities
 import akka.actor.ActorSystem
 import akka.http.scaladsl.model.StatusCodes
+
+import scala.collection.JavaConversions._
```

Comment:
This is actually deprecated in scala 2.12 in favor of `JavaCoverters`. I think we should remove this import -- you might need to add `.asJava` more places below.